src/impl.jl

```diff
@@ -234,7 +234,7 @@ function auto_hash_equals_impl(__source__, struct_decl, fields, cache::Bool, has
             if typearg
                 :($type_seed($full_type_name))
             else
-                Base.hash(type_name)
+                :($hashfn($(QuoteNode(type_name))))
```

Comment:
These changes give the backward-compatible hash codes from 1.0 that `raicode` depends on.